reV/SAM/SAM.py

```diff
@@ -65,7 +65,37 @@ def _get_base_handler(res_file, module):
         return res_handler
 
     @staticmethod
-    def _make_solar_kwargs(res_handler, project_points, output_request):
+    def _parse_gid_map_sites(gen_gids, gid_map):
+        """Parse resource gids based on the generation gids used by
+        project_points and a gid_map. If gid_map is None, the input gen_gids
+        are just passed through as the res_gids.
+
+        Parameters
+        ----------
+        gen_gids : list
+            List of project_points "sites" that are the generation gids.
+        gid_map : None | dict
```

Comment:
Do we want to set this as a kwarg with default None? I find that to be more intuative than having the possible dtypes be None | dict

Reply:
Yeah that might be easier to read. The only reason i didnt is because its a private method that should always be called with both arguments. Calling the method without gid_map is pointless so i wanted to make sure it gets input.

Reply:
Yep agreed on the readability. will do!